normandy/recipes/exports.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+import kinto_http
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.functional import cached_property
+
+
+logger = logging.getLogger(__name__)
+
+REMOTE_SETTINGS_BUCKET = "main-workspace"
+
+
+class RemoteSettings(object):
+    """
+    Interacts with a RemoteSettings service.
+
+    """
+    def __init__(self):
+        self.check_config()
+        self.cid = settings.REMOTE_SETTINGS_COLLECTION_ID
+
+    @cached_property
```

Comment:
`cached_property` [stores the result on `self`](https://docs.djangoproject.com/en/2.1/ref/utils/#django.utils.functional.cached_property). Since you only ever use this by something like `RemoteSettings().publish()`, the cache isn't going to help at all. Perhaps it should be cached in a module level variable? What is the cost of recreating a Kinto client?